agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -229,9 +233,9 @@ private static void extracted(DiseaseAnnotationDocument annotation, org.alliance
 		DataProvider dataProvider = primaryAnnotation.getDataProvider();
 		if (dataProvider != null && dataProvider.getCrossReference() != null) {
 			String urlTemplate = dataProvider.getCrossReference().getResourceDescriptorPage().getUrlTemplate();
-			if(urlExpceptionHandler.contains(dataProvider.getSourceOrganization().getAbbreviation())){
+			if (urlExpceptionHandler.contains(dataProvider.getSourceOrganization().getAbbreviation())) {
 				// remove the prefix in the template as the prefix is already in the curie.
-				urlTemplate = urlTemplate.replace(dataProvider.getSourceOrganization().getAbbreviation()+":","");
+				urlTemplate = urlTemplate.replace(dataProvider.getSourceOrganization().getAbbreviation() + ":", "");
 			}
```

Comment:
this code needs to be in the url template object basically pass the id in and get a url back you will need this in multiple places

Reply:
I agree, but that has to happpen in the curation repo first until it can get used here.
I'd like to get his out before sprint review.